src/schema/tags.ts

```diff
@@ -76,5 +110,48 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (source, args, ctx): Promise<GQLTagResults> => {
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        where: {
+          flags: {
+            onboarding: true,
+          },
+        },
+        order: { value: 'ASC' },
+      });
+
+      return {
+        hits: hits.map((hit) => ({ name: hit.value })),
+      };
+    },
+    recommendedTags: async (
+      source,
+      { tags, excludedTags },
+      ctx,
+    ): Promise<GQLTagResults> => {
+      const uniqueTagsToExclude = new Set([...tags, ...excludedTags]);
+
+      if (uniqueTagsToExclude.size > 1000) {
+        throw new ValidationError('Tag limit reached');
```

Comment:
Is this passed through to the client as is, or does it require special handling? If it does require, we might change this message to a code from `errors.ts`

Reply:
Good point, added in 16581ff6a1a70cbdf2c814d6523cd64d5444b3bf